voidcrew/code/datums/ruins/beach.dm

```diff
@@ -0,0 +1,13 @@
+/datum/map_template/ruin/beach/fishinghut
```

Comment:
make a generic `/datum/map_template/ruin/beach/` and put the prefix there, so you don't have to include it with every single ruin